src/style.css

```
@@ -0,0 +1,138 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
```

You can combine them as follow:
```suggestion
body,
p {
  margin: 0;
  padding: 0;
}
```